build/build-sdk-images/restapi/gen.sh

```diff
@@ -14,7 +14,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-mkdir /go/src/agones.dev/agones/swagger
 wget -q https://repo1.maven.org/maven2/io/swagger/swagger-codegen-cli/2.4.10/swagger-codegen-cli-2.4.10.jar -O /tmp/swagger-codegen-cli.jar
```

Comment:
(non blocking question): Any desire to upgrade the swagger-codegen-cli ? No idea how much it matters.

Looks like this version came out in 2019

[2.4.10/](https://repo1.maven.org/maven2/io/swagger/swagger-codegen-cli/2.4.10/)     2019-11-16 16:38

Reply:
Updated to v3.0.35(latest)